ktxspec.adoc

```diff
@@ -1918,6 +1947,36 @@ seconds is stem:[duration / timescale].
 This metadata entry must not be used together with
 <<KTXcubemapIncomplete,`KTXcubemapIncomplete`>>.
 
+=== KTXmapRange
+When data is input from a source with 32-bit floating point data,
+such as a .exr file, and compressed to an HDR block format, such
+as ASTC HDR, BC6H or UASTC HDR, the values in the source may be
+mapped to the more limited range of the block-compressed format.
+KTX file writers may indicate the mapping by using the key
+
+- `KTXmapRange`
+
+The value is 8 bytes representing 2 IEEE 754 Float32 values:
+[source,c]
+----
+Float32 scale
+Float32 offset
+----
+
+A shader can restore the original 32-bit float value with
+[stem]
++++++
+effective = sampled \times scale + offset
++++++
+
+[IMPORTANT]
+====
+It's important to multiply first so that decoders can use a single
+fused multiply-add instruction to apply the scale and the offset.
+====
```

Comment:
Since the exact decoding expression is given above, this note is not needed. Besides it's an application choice to use FMA or not because it may have platform-specific precision differences.

Reply:
This and other KTXmapRange related comments is addressed in commit 3825119.

Thanks for the excellent suggestions.